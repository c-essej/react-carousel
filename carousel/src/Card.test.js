import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";



it("renders without crashing", function () {
  render(<Card
      caption="testcaption"
      src="test1.com"
      currNum="2"
      totalNum="4"/>)
});

it("matches snapshot", function () {
  const { container } = render(<Card
      caption="testcaption"
      src="test1.com"
      currNum="2"
      totalNum="4"/>);
  expect(container).toMatchSnapshot();
})