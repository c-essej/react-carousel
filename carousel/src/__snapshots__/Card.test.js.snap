// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testcaption
    </h4>
    <img
      alt="testcaption"
      class="Card-image"
      src="test1.com"
    />
    <small
      class="Card-small"
    >
      Image 
      2
       of 
      4
      .
    </small>
  </div>
</div>
`;
